src/UserManager.js

```diff
@@ -93,13 +93,11 @@ export class UserManager extends OidcClient {
     }
     signinRedirectCallback(url) {
         return this._signinEnd(url || this._redirectNavigator.url).then(user => {
-            if (user) {
-                if (user.profile && user.profile.sub) {
-                    Log.info("UserManager.signinRedirectCallback: successful, signed in sub: ", user.profile.sub);
-                }
-                else {
-                    Log.info("UserManager.signinRedirectCallback: no sub");
-                }
+            if (user.profile && user.profile.sub) {
```

Comment:
The null check here was unnecessary and misleading. `this._signinEnd()` is guaranteed to resolve to a non-null `User` because of how it is implemented to unconditionally create/return a new instance of `User`.